Jenkinsfile

```diff
@@ -1,17 +1,101 @@
+void CreateCluster() {
+    withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-key-file', variable: 'CLIENT_SECRET_FILE')]) {
+        sh '''
+            source $HOME/google-cloud-sdk/path.bash.inc
+            gcloud auth activate-service-account --key-file $CLIENT_SECRET_FILE
+            gcloud config set project $GCP_PROJECT
+            gcloud container clusters create --zone us-central1-a $CLUSTER_NAME-$CLUSTER_PREFIX --cluster-version 1.12 --machine-type n1-standard-4 --preemptible --num-nodes=3
+            kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user jenkins@$GCP_PROJECT.iam.gserviceaccount.com
+        '''
+   }
+}
+void pushArtifactFile(String FILE_NAME) {
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+        sh """
+            S3_PATH=s3://percona-jenkins-artifactory/\$JOB_NAME/\$(git rev-parse --short HEAD)
+            aws s3 ls \$S3_PATH/${FILE_NAME} || :
```

Comment:
which line install aws-cli?

Reply:
It is installed by default on this instance.